proto/axelar/vote/exported/v1beta1/types.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+package axelar.vote.exported.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/vote/exported";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "axelar/utils/v1beta1/threshold.proto";
+import "cosmos_proto/cosmos.proto";
+import "axelar/snapshot/exported/v1beta1/types.proto";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// PollKey represents the key data for a poll
+message PollKey {
+  option (gogoproto.goproto_stringer) = false;
+
+  string module = 1;
+  string id = 2 [ (gogoproto.customname) = "ID" ];
+}
+
+message Voter {
+  bytes validator = 1 [ (gogoproto.casttype) =
+                            "github.com/cosmos/cosmos-sdk/types.ValAddress" ];
+  int64 voting_power = 2;
+}
+
+message Vote { google.protobuf.Any result = 1; }
+
+// PollMetadata represents a poll with write-in voting, i.e. the result of the
+// vote can have any data type
+message PollMetadata {
+  vote.exported.v1beta1.PollKey key = 1 [ (gogoproto.nullable) = false ];
+  int64 expires_at = 3;
+  google.protobuf.Any result = 4
+      [ (cosmos_proto.accepts_interface) =
+            "github.com/cosmos/codec/ProtoMarshaler" ];
+  utils.v1beta1.Threshold voting_threshold = 5 [ (gogoproto.nullable) = false ];
+  exported.v1beta1.PollState state = 6;
+  int64 min_voter_count = 7;
+  repeated Voter voters = 8 [ (gogoproto.nullable) = false ];
+  bytes total_voting_power = 9 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string reward_pool_name = 10;
+}
+
+enum PollState {
+  option (gogoproto.goproto_enum_prefix) = false;
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  // these enum values are used for bitwise operations, therefore they need to
+  // be powers of 2
+  POLL_STATE_UNSPECIFIED = 0
+      [ (gogoproto.enumvalue_customname) = "NonExistent" ];
+  POLL_STATE_PENDING = 1 [ (gogoproto.enumvalue_customname) = "Pending" ];
+  POLL_STATE_COMPLETED = 2 [ (gogoproto.enumvalue_customname) = "Completed" ];
+  POLL_STATE_FAILED = 4 [ (gogoproto.enumvalue_customname) = "Failed" ];
+  POLL_STATE_EXPIRED = 8 [ (gogoproto.enumvalue_customname) = "Expired" ];
+  POLL_STATE_ALLOW_OVERRIDE = 16
+      [ (gogoproto.enumvalue_customname) = "AllowOverride" ];
+}
```

Comment:
this file just moved + has the namespace added, git just isn't able to match it with the original, because that one still exists for the migration